fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -737,7 +737,7 @@ void SimulationConfig::generateNormalConfig(int minimumReplication, int minimumR
 	if (deterministicRandom()->random01() < 0.25) db.desiredTLogCount = deterministicRandom()->randomInt(1,7);
 	if (deterministicRandom()->random01() < 0.25) db.masterProxyCount = deterministicRandom()->randomInt(1,7);
 	if (deterministicRandom()->random01() < 0.25) db.resolverCount = deterministicRandom()->randomInt(1,7);
-	int storage_engine_type = deterministicRandom()->randomInt(0, 3);
+	int storage_engine_type = 3;//deterministicRandom()->randomInt(0, 3);
```

Comment:
I started a correctness run on the state of this PR, so it's good this change is here for now, but we'll have to remove it prior to merge.

Reply:
Next commit reverts this code.